addons/website/static/src/js/menu/edit.js

```diff
@@ -72,9 +72,18 @@ var EditPageMenu = websiteNavbarData.WebsiteNavbarActionWidget.extend({
     start: function () {
         var def = this._super.apply(this, arguments);
 
+        let navbar = $('nav#oe_main_menu_navbar').get(0);
+        let stopPropagation = function (event) {
```

Comment:
- const
- move it inside the if, don't define it outside if you don't need it
- probably es6 arrow